docs/roman/datamodels/models.rst

```diff
@@ -8,58 +8,84 @@ the underlying file format.  The same data model may be used for data
 created from scratch in memory, or loaded from ASDF files or some future file
 format.
 
+The detailed datamodel structure and specifics are contained in the
+documentation included with the roman_datamodels package found here (TBD).
 
-Hierarchy of models
--------------------
-
-There are different data model classes for different kinds of data.
-
-One model instance, many arrays
--------------------------------
-
-Each model instance generally has many arrays that are associated with
-it.  For example, the `ImageModel` class has the following arrays
-associated with it:
+Each model instance is created to contain a variety of attributes and data that
+are for needed for analysis or to propogate information about the file and the
+contents of the file. For example, the `ImageModel` class has the following
+arrays associated with it:
 
     - `data`: The science data
     - `dq`: The data quality array
     - `err`: The error array
 
-The shape of these arrays must be broadcast-compatible.  If you try to
-assign an array to one of these members that is not
-broadcast-compatible with the data array, an exception is raised.
+The shape of these arrays must be broadcast-compatible. If you try to
+assign an array to one of these members that is not broadcast-compatible with
+the data array, an exception is raised.
```

Comment:
I can see why this is useful but is there a check for this, i.e. is an exception raised?

Reply:
I tested this and didn't see an error.  Must be leftover documentation from jwst?

Reply:
Actually I am wrong, it does not throw an error unless I try to use the arrays. It will save and verify with arrays that are not broadcast compatible. Another ticket to write?